tools/distpartitioning/README.md

```diff
@@ -0,0 +1,40 @@
+### xxx_nodes.txt format
+This file is used to provide node information to this framework. Following is the format for each line in this file:
+```
+<node_type> <weight1> <weight2> <weight3> <weight4> <global_type_node_id> <attributes>
+```
+where node_type is the type id of this node, weights can be any number of columns as determined by the user, type_node_id is the `locally` contiguous ids starting from `0` for a particular node_type. And attributes can be any number of columns at the end of each line. 
+
+###xxx___edges.txt format
+This file is used to provide edge information to this framework. Following is the format for each line in this file:
+```
+<global_src_id> <global_dst_id> <global_type_edge_id> <edge_type> <attributes>
+```
```

Comment:
the same question as above. why is the format here different from the format supported by ParMETIS?
are these file passed to ParMETIS?

Reply:
you didn't address my comment.

Reply:
As discussed in the code review meeting a couple of days ago, this is the correct format. The formats for inputs and outputs of ParMETIS are different.